.github/workflows/deploy.yml

```diff
@@ -53,12 +46,57 @@ jobs:
             cd go-tableland
             git fetch && git checkout -f ${{ github.sha }}
             cd docker
-            
-            HEALTHBOT_ENABLED=true make ${{ steps.prep.outputs.deployment }}-up
+
+            HEALTHBOT_ENABLED=true make staging-up
 
             # Wait to start and double-check we're running the expected version.
             # This helps having a green check in the GH Action be high-signal that everything is fine.
             sleep 5;
             for i in 1 2 3 4; do [ $(curl --insecure --silent https://127.0.0.1/version | jq .git_commit) = "\"${{ steps.prep.outputs.sha_short}}\"" ] && break || (if [ $i = 4 ]; then exit -1; else sleep 5; fi ) done 
             echo "All healthy!"
 
+      - if: github.ref == 'refs/heads/main'|| contains(github.event.head_commit.message, '[testnetdeploy]')
+        name: Deploy testnet
+        uses: appleboy/ssh-action@master
+        with:
+          host: ${{ secrets.TESTNET_V2_VM_HOST }}
+          username: validator
+          key: ${{ env.vm_ssh_key }}
+          port: 22
+          script_stop: true
+          command_timeout: 30m
+          script: |
+            cd go-tableland
+            git fetch && git checkout -f ${{ github.sha }}
+            cd docker
+
+            make testnet-up
```

Comment:
Temporarily I removed the healthbot for this new testnet validator. We should avoid having two healthbots enabled for the same environment since that messes up the expectations of the healthbot logic.
We can re-enable it again as soon as this becomes the main testnet validator again.